src/app/overview/overview.component.ts

```diff
@@ -269,12 +269,15 @@ export class OverviewComponent extends SubscriptionManager implements OnInit {
           map((results) => {
             const qp = results[1];
             const qpValArrays = {};
-
             Object.keys(qp).forEach((paramName: string) => {
               this.queryParamsRaw[paramName] = [];
               qpValArrays[paramName] = (
                 Array.isArray(qp[paramName]) ? qp[paramName] : [qp[paramName]]
               ).map((qpVal: string) => {
+                qpVal =
```

Comment:
What is `qpVal`? Please write a name that makes it clearer what is supposed to be

Reply:
Renamed